build/lib/release.sh

```diff
@@ -344,6 +344,8 @@ function kube::release::create_docker_images_for_server() {
     local DOCKER_BUILD_OPTS=()
     if [[ "${KUBE_BUILD_PULL_LATEST_IMAGES}" =~ [yY] ]]; then
         DOCKER_BUILD_OPTS+=("--pull")
+    else
+        DOCKER_BUILD_OPTS+=("")
```

Comment:
This should not be needed, as `DOCKER_BUILD_OPTS` is defined on L344

Reply:
Thanks for your reply. But I tested multiple-times and i still  get the `unbound variable` error if not with `DOCKER_BUILD_OPTS+=("")`.

Reply:
https://stackoverflow.com/a/7577209

It depends on the bash version

Reply:
instead we should probably expand DOCKER_BUILD_OPTS in a way that doesn't cause this error